xbmc/video/ContextMenus.h

```diff
@@ -20,7 +20,7 @@ class CVideoInfo : public CStaticContextMenuAction
 public:
   explicit CVideoInfo(MediaType mediaType);
   bool IsVisible(const CFileItem& item) const override;
-  bool Execute(const CFileItemPtr& item) const override;
+  bool Execute(const std::shared_ptr<CFileItem>& item) const override;
```

Comment:
Please add `#include <memory>`to this header.

Reply:
done